nvfuser/__init__.py

```diff
@@ -162,7 +163,18 @@ def execute(
                             f".as_strided({tuple(i.size())}, {tuple(i.stride())}),\n"
                         )
                 else:
-                    msg += f"    {i},\n"
+                    input_as_string = str(i)
```

Comment:
I'm wondering how this hasn't shown up in our test with extreme values yet.

nitpick, can we add a test case to verify this?

Reply:
> I'm wondering how this hasn't shown up in our test with extreme values yet.

I thought we never auto-test reproducers generated here? 

> nitpick, can we add a test case to verify this?

Maybe. Something like 
1. run a known bad program which generates a reproducer
2. extract the reproducer from stdout
3. run the reproducer to see if it gives the same error
?

Reply:
oh nvm. this is patching the repro scripts. I wasn't paying enough attention here and thought it's the stringification thing from fusion definition. My bad.

Reply:
no need to add test here. Let's merge it.